server/mailet/integration-testing/src/test/java/org/apache/james/mailets/DirectResolutionRemoteDeliveryIntegrationTest.java

```diff
@@ -56,13 +56,19 @@
 
 import com.google.common.collect.ImmutableList;
 
+import io.restassured.response.ValidatableResponse;
+
 public class DirectResolutionRemoteDeliveryIntegrationTest {
     private static final String JAMES_ANOTHER_DOMAIN = "james.com";
     private static final String JAMES_ANOTHER_MX_DOMAIN_1 = "mx1.james.com";
     private static final String JAMES_ANOTHER_MX_DOMAIN_2 = "mx2.james.com";
     private static final List<String> JAMES_ANOTHER_MX_DOMAINS = ImmutableList.of(JAMES_ANOTHER_MX_DOMAIN_1, JAMES_ANOTHER_MX_DOMAIN_2);
 
     private static final String FROM = "from@" + DEFAULT_DOMAIN;
+    public static final Function<ValidatableResponse, ValidatableResponse> TESTMAIL_RECEIVED_EXPECTATION = response -> response
```

Comment:
how does it help for stability ?

Reply:
Yes. Awaitability dare retry using untilAsserted stuff. Not with until stuff. No Idea why.